user_guide_src/source/general/alternative_php.rst

```diff
@@ -0,0 +1,75 @@
+###################################
+Alternate PHP Syntax for View Files
+###################################
+
+If you do not utilize CodeIgniter's :doc:`template
+engine <../libraries/parser>`, you'll be using pure PHP in your
+View files. To minimize the PHP code in these files, and to make it
+easier to identify the code blocks it is recommended that you use PHPs
+alternative syntax for control structures and short tag echo statements.
+If you are not familiar with this syntax, it allows you to eliminate the
+braces from your code, and eliminate "echo" statements.
+
+Automatic Short Tag Support
+===========================
+
+.. note:: If you find that the syntax described in this page does not
```

Comment:
I don't believe this paragraph is needed. The only short tag we describe is the short echo, which is always enabled as of PHP 5.4, I believe, but definitely in 7. And there doesn't currently exist any way to have short tags automatically converted.